pkg/apis/miniojobs.min.io/register.go

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2022, MinIO, Inc.
+//
+// This code is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License, version 3,
+// as published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License, version 3,
+// along with this program.  If not, see <http://www.gnu.org/licenses/>
+
+package operator
+
+// MinIO Operator MinIOJobs group name.
+const (
+	GroupName = "miniojobs.min.io"
```

Comment:
We can use the same GroupName as `Tenant` `minio.min.io` since this will be MinIO specific